From 9551ff1c56515cf0d9a223df6b4dd3edb5727d22 Mon Sep 17 00:00:00 2001
From: Wang Hui <Hui.Wang@windriver.com>
Date: Wed, 24 Apr 2013 17:25:10 +0800
Subject: [PATCH 021/158] drivers/i2c/ai2c: remove default y from Kconfig

If we set default y for this driver in the Kconfig, it will be built
when we build other platforms, it will bring building errors, so
remove it.

Signed-off-by: Wang Hui <Hui.Wang@windriver.com>
---
 drivers/i2c/busses/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/i2c/busses/Kconfig b/drivers/i2c/busses/Kconfig
index 1eee27b..0166fb0 100644
--- a/drivers/i2c/busses/Kconfig
+++ b/drivers/i2c/busses/Kconfig
@@ -956,7 +956,6 @@ config SCx200_ACB
 config I2C_AXXIA
 	tristate "Axxia I2C bus support"
 	select I2C_ALGOBIT
-	default y
 	help
 	  Say yes if you want to support the I2C bus on Axxia platforms.
 
-- 
1.8.1.2

